CHANGELOG.md

```diff
@@ -109,6 +109,10 @@
   [#9962](https://github.com/Kong/kong/pull/9962)
 - **OpenTelemetry**: Support scoping with services, routes and consumers.
   [#10096](https://github.com/Kong/kong/pull/10096)
+- **Statsd**: Add `tag_style` configuration
+  parameter that allows statsd send metrics by tag.
+  Defaults to `nil`
```

Comment:
Defaults to `nil` *which means xxxxx.*

Should be clearer on what this option actually does to make the changelog entry useful for our users.